.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Issue template
+about: Questions and enhancement requests should go to the forum.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+
+**Code to reproduce the issue**
+
+**Expected Results**
+
+**Actual Results**
+
+**ESP32 Chip version**
+
+**ESP-IDF version**
+
+**Development kit used**
+
+**Development machine OS**
+
+**Compilation warnings/errors (if available)**
+
+**If possible, copy the compilation log into a file and attach it here**
+
+
+We use GitHub issues for development related discussions.
```

Comment:
I'd move it to the top to increase the chance that people will read it.

Reply:
Done in 7e5deae